NuGet.config

```diff
@@ -4,7 +4,7 @@
 <configuration>
   <packageSources>
     <clear />
-    <add key="myget.org roslyn-master-nightly" value="https://dotnet.myget.org/F/roslyn-master-nightly/api/v3/index.json" />
+    <add key="myget.org roslyn" value="https://dotnet.myget.org/F/roslyn/api/v3/index.json" />
```

Comment:
you may be right, just checked, thatz the url which was provided for dogfooding as well... not sure why manish had different url.